msteams-platform/tabs/how-to/create-channel-group-tab.md

```diff
@@ -1028,9 +1028,7 @@ You've completed the tutorial to build a tab app with Blazor.
 
 > [!NOTE]
 >
-> * Migrating your configurable tab to static tab is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
-> * Migrating your configurable tab to static tab is available only in classic Teams client and isn't available in the [new Teams client](~/resources/teams-updates.md). 
-> * To migrate your configurable tab to static tab, use the app manifest v1.16 or later.
+> To migrate your configurable tab to static tab, use the app manifest v1.16 or later.
 
 Static tab capability is extended to support group chat, channels, and meetings. You can update your existing configurable tab to static tab and add different scopes to the static tab. 
```

Comment:
Can we add a little more reasoning why developer should consider static tabs over configurable tabs? ex:
> Static tab capability is extended to support group chat, channels, and meetings. **With static tabs, the configuration dialog is skipped allowing users to instantly pin the tab. Light configuration can still happen in the tab itself after it's been pinned since the `setConfig` APIs will allow you to change the `contentUrl`.** You can update your existing configurable tab to static tab and add different scopes to the static tab.

Reply:
Is addressed through PR #10514